recipes/pulsar_phase/pulsar_phase_computation.ipynb

```diff
@@ -0,0 +1,1895 @@
+{
```

Comment:
<p>this suggests that this nb can work with any version of Gammapy. And I think that this could be not true on the long term.</p><p>Instead of X.Y.Z, precise explicitly the used version of Gammapy</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy-recipes/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>